### PR TITLE
Remove Guava dependency from examples

### PR DESCRIPTION
**Pull request by minwoox**
We have decided to minimize the use of 3rd party depdendencies in examples.
Guava is one of them.

### CODE REVIEW

File: examples/context-propagation/manual/build.gradle [hunk 1]
```diff
@@ -4,6 +4,7 @@ plugins {
 
 dependencies {
     implementation project(':core')
+    implementation 'com.spotify:completable-futures'
```
Comment (trustin) [inline examples/context-propagation/manual/build.gradle:7]: Is it fine to pull this in? Should we remove this dependency as well?

Comment (minwoox) [reply examples/context-propagation/manual/build.gradle:7]: I tried to make the `context-propagation/dagger` example without it, but it doesn't look good as it is.
https://github.com/line/armeria/blob/master/examples/context-propagation/manual/src/main/java/example/armeria/contextpropagation/manual/MainService.java#L93

So I can use the dependency in this case, but if it's a problem, let me know. I will remove it. 😄

Comment (trustin) [reply examples/context-propagation/manual/build.gradle:7]: No worries, then. 😄

**Comment by trustin** [comment]
Thanks, @minwoox !

**Pull request merged**